src/dsp_tools/xmllib/models/file_values.py

```diff
@@ -60,11 +62,13 @@ def __post_init__(self) -> None:
             _warn_type_mismatch(expected_type="IIIF uri", value=self.value, res_id=self.resource_id)
 
     def serialise(self) -> etree._Element:
-        attribs = {}
-        if self.permissions != Permissions.PROJECT_SPECIFIC_PERMISSIONS:
-            attribs["permissions"] = self.permissions.value
-        if self.comment:
-            attribs["comment"] = self.comment
+        attribs = (
+            {"permissions": self.permissions.value}
+            if self.permissions != Permissions.PROJECT_SPECIFIC_PERMISSIONS
+            else {}
+        )
```

Comment:
I think the old syntax is easier to read:

```suggestion
        attribs = {}
        if self.permissions != Permissions.PROJECT_SPECIFIC_PERMISSIONS:
            attribs["permissions"] = self.permissions.value
```